docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -338,6 +339,11 @@ const Experimental = () => {
 };
 
 const Pre = ({children, ...props}) => {
+  // Retrieve the DAG image to display next to the code block, if any
```

Comment:
It would be nice to have real props defined for this component, even if it's just `React.HTMLProps<HTMLPreElement>`.